code/API_definitions/Application-endpoint-registration.yaml

```diff
@@ -0,0 +1,392 @@
+openapi: 3.0.3
+info:
+  title: Application Endpoint Registration API
+  version: 0.1.0
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  description: |
+    Application endpoint registration allows application developers to
+    register one or more Application Endpoints, and retrieve, update,and
```

Comment:
Should we have a definition for "Application Endpoints" to make it more explicit?

Reply:
I also think that these Application Endpoints are not generic and they corresponds to the application instances running on edge cloud zones in probably distributed manner. So should we qualify the Application Endpoints in some way to indicate developers about the expectations from these endpoints i.e. edge instances?